### ISSUE TITLE
Downsides of forking 3rd party repositories here

### ISSUE THREAD

**Issue by mschubert** [Snakemake-Profiles/doc#5]
If a repository is forked from another source, its issues will be disabled by default.

If I then report an issue with the profile to the original source, this will not be automatically propagated to this collection.

Wouldn't it be better to move repositories here instead of forking them? (and only forking if they change)

Authors would still get credit (because they are listed as commiters)

(Anyway, this is just a suggestion so feel free to ignore)

**Comment by johanneskoester** [comment]
Mhm, wait, we have issues for the pbs-torque repo, which is forked...

**Comment by johanneskoester** [comment]
I have to enable them. Now we also have them for slurm. Thanks for making me aware of this!

**Comment by mschubert** [comment]
Yes, then the question is if you want to have separate development workflows or one.

Example:

I sent a PR to https://github.com/percyfal/slurm to add more detailed installation instructions. Now you fixed them here, which is great, but you'll have a conflict if the original repo accepts the PR.

This is why I thought it would make most sense to ask original maintainers if they agree with moving repos to the Snakemake-Profiles user.

**Comment by johanneskoester** [comment]
Yes, we can do that as well, you are right.
